src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,95 @@
+package api.requests;
+
+import api.support.APITests;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import api.support.http.ResourceClient;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.domain.ItemStatus;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class ItemsInTransitReportTests extends APITests {
+
+  private static final String LOCATION_NAME = "name";
+  private static final String LOCATION_CODE = "code";
+  private static final String LIBRARY = "library";
+  private static final String STATUS_KEY = "status";
+  private static final String BARCODE_KEY = "barcode";
+  private static final String TITLE = "title";
+  private static final String CONTRIBUTORS = "contributors";
+  private static final String DESTINATION_SERVICE_POINT = "inTransitDestinationServicePointId";
+
+  @Test
+  public void reportIsEmptyWhenThereAreNoItemsInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    List<JsonObject> items = ResourceClient.forInventoryReport(client).getAll();
+
+    assertTrue(items.isEmpty());
+  }
+
+  @Test
+  public void reportWhenThereAreItemInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final JsonObject smallAngryPlanetInstance = smallAngryPlanet.getInstance().getJson();
+    final String contributors = String.valueOf(((JsonArray) smallAngryPlanetInstance
+      .getMap().get(CONTRIBUTORS)).getJsonObject(0).getMap().get("name"));
+
+    // init for SP2
+    final IndividualResource steve = usersFixture.steve();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    //#1 checkout SP 1
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+
+    // #2 create the request in SP2
+    RequestBuilder secondRequestBuilderOnItem = new RequestBuilder()
+      .open()
+      .hold()
+      .withPickupServicePointId(secondServicePointId)
+      .forItem(smallAngryPlanet)
+      .by(steve);
+    requestsClient.create(secondRequestBuilderOnItem);
+
+    // #3 check-in item in SP1
+    loansFixture.checkInByBarcode(smallAngryPlanet);
```

Comment:
Does it make sense to extract this logic to a separate method and name it e.g. createRequestsAndStartFylfillment

Reply:
This code is used only in one place and it seems to me it is more readable what exact steps do in approach like this.

Reply:
What do the comments tell the reader that the code does not? 

Are they there because you find the code is not clear enough in its intent?